ee/query-service/app/api/featureFlags.go

```diff
@@ -22,3 +38,62 @@ func (ah *APIHandler) getFeatureFlags(w http.ResponseWriter, r *http.Request) {
 	}
 	ah.Respond(w, featureSet)
 }
+
+// fetchZeusFeatures makes an HTTP GET request to the /zeusFeatures endpoint
+// and returns the FeatureSet.
+func fetchZeusFeatures(url string) (basemodel.FeatureSet, error) {
```

Comment:
Let's put the SigNoz-Cloud Auth here!
You can find references here: https://github.com/SigNoz/signoz/blob/7811fdd17a00e474ee354c7b3760bbb622dcc19a/ee/query-service/app/api/gateway.go#L29C2-L29C54

```
req.Header.Set("X-Signoz-Cloud-Api-Key", licenseKey)
```